### ISSUE TITLE
Remove error thrown if no json files are found

### ISSUE THREAD

**Issue by wswebcreation** [wswebcreation/multiple-cucumber-html-reporter#54]
Now the module throws an issue when no files are found. This blocks the `protractor-multiple-cucumber-html-reporter-plugin`.

https://github.com/wswebcreation/protractor-multiple-cucumber-html-reporter-plugin/issues/14#issuecomment-397968207

**Comment by daviddekanter** [comment]
Fixed with this PR:
https://github.com/wswebcreation/multiple-cucumber-html-reporter/pull/55

**Comment by vignesh-paramasivam** [comment]
Thanks for fixing. When this will be updated as npm package? Waiting for this :)

**Comment by wluu** [comment]
@wswebcreation 

I have a very similar setup as @vignesh-paramasivam  and I see the fix for this issue (using `protractor-multiple-cucumber-html-reporter-plugin@1.8.0`). But now, I'm getting these runtime errors on ignored features:

```
WARNING: No JSON files found in '/myproject/test/bdd/report'. NO REPORT CAN BE CREATED!
I/plugins -  	 Fail:  protractor-multiple-cucumber-html-reporter-plugin Runtime 
E/plugins - 		Failure during postResults: suite.features.forEach is not a function
E/plugins - 		TypeError: suite.features.forEach is not a function
 		at _parseFeatures (/myproject/node_modules/multiple-cucumber-html-reporter/lib/generate-report.js:154:20)
```

Should I open another issue?

**Comment by wswebcreation** [comment]
Hmm,

This should not happen. Can you check your package-lock.json and see what version of multiple-cucumber-html-reporter is used there?

**Comment by wluu** [comment]
Can do:
```
$ cat package-lock.json | jq '.dependencies["multiple-cucumber-html-reporter"].version'
"1.11.3"
```

**Comment by wluu** [comment]
I think this is the issue: https://github.com/wswebcreation/multiple-cucumber-html-reporter/blob/master/lib/collect-jsons.js#L55

**Comment by wluu** [comment]
I made a PR for this little bug: https://github.com/wswebcreation/multiple-cucumber-html-reporter/pull/71

**Comment by wswebcreation** [comment]
Tnx, just released version `1.11.4`
